Doc/library/ssl.rst

```diff
@@ -215,6 +215,25 @@ purposes.
       :data:`VERIFY_X509_STRICT` in its default verify flags.
 
 
+Signature algorithms
+^^^^^^^^^^^^^^^^^^^^
+
+.. function: get_sigalgs()
```

Comment:
```suggestion
.. function:: get_sigalgs()
```

With only two leading dots and a single colon after function, the entire paragraph is actually... a comment :') I'll actually open an issue for the following follow-up (docs) works:

- Indicate positional-only parameters in SSL docs.
- Indicate which OpenSSL version is required for the functionality.
- Reword some English if needed (I don't know if it's needed but maybe).

Do you remember if there were more things that I needed to do on the docs?

Reply:
We also discussed changing `_ssl__SSLContext_set_ciphers_impl` to use `_setSSLError` similar to what I did in `_ssl__SSLContext_set_ciphersuites_impl`. There might be other places manually doing `ERR_clear_error()` which could benefit from this. As part of that, the errcode argument can be removed from that function, as the value passed in is never used.

You also mentioned something about another PR which was updating the "Contributed by" lines to consistently include a period at the end).